src/main/java/duke/tasks/Deadlines.java

```diff
@@ -0,0 +1,28 @@
+package duke.tasks;
+
+import duke.Task;
+
+public class Deadlines extends Task {
+    protected String date;
```

Comment:
Private would be better since no classes extends from Deadlines and you already have a getter method for it.